### DISCUSSION TITLE
My document disappeared without an updated version

### DISCUSSION THREAD

**Discussion by zhinianyuxin0o0** [outline/outline#7864] [Self Hosting]
I was running version 0.78.0 of outline on my own server, but recently the documentation suddenly stopped showing and I did not make any version updates,  I recovered the document by importing the backup, but I wonder what happened?

**Comment by linear** [comment]
<p><a href="https://linear.app/outline/issue/OLN-729/my-document-disappeared-without-an-updated-version">OLN-729 My document disappeared without an updated version</a></p>
